src/main/java/frc/robot/subsystems/intake/IntakeRollerConstant.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.subsystems.intake;
+
+public class IntakeRollerConstant {
+
+	public static final String LOG_PATH = "Subsystems/IntakeRoller/";
```

Comment:
```suggestion
	public static final String LOG_PATH = "IntakeRoller/";
```

By default, GB subsystem is logged in the Subsystems folder